examples/surfaces.html

```diff
@@ -106,6 +106,10 @@
   });
   document.body.appendChild(arDebug.getElement());
 
+  vrDisplay.addEventListener('vrdisplayplanesadded', (event) => {
```

Comment:
Just a reminder that this event type has changed to  "planesadded"

Reply:
In addition, this listener should be removed because the surfaces functionality is coming from arDebug above. Consider reverting this file?